### PR TITLE
Cat 1185

### PR DESCRIPTION
**Pull request by Olusola-Opencast**
Added more information to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,46 +22,74 @@
 [![Postgres](https://img.shields.io/badge/postgres-%23316192.svg?style=postgres&logo=postgresql&logoColor=white)](https://www.postgresql.org/)
 [![Redis](https://img.shields.io/badge/redis-%23DD0031.svg?style=flat&logo=redis&logoColor=white)](https://redis.io/)
 
-# Dev Website
 
-https://dev.offender-categorisation.service.justice.gov.uk/
```
Comment (kberec) [inline README.md:27]: I think it would be still useful to keep it in, maybe to add pre-prod/prod urls as well?

File: README.md [hunk 2]
```diff
@@ -22,46 +22,74 @@
...
+|              hmpps-auth               | OAuth2 API server for authenticating requests          | http://localhost:9090/auth                                              | NOMIS_AUTH_EXTERNAL_URL<br>NOMIS_AUTH_URL                                                 |
+|             risk-profiler             | Risk Profiler service                                  | http://localhost:8082/                                                  | RISK_PROFILER_ENDPOINT_URL                                                                |
+|          allocation-manager           | Allocation manager                                     | http://localhost:8083/                                                  | ALLOCATION_MANAGER_ENDPOINT_URL                                                           |
+|               postgres                | PostgreSQL database server for offender-categorisation | psql://localhost:5432                                                   | None - required locally                                                                   |
```
Comment (kberec) [inline README.md:51]: "None - required locally" - let's discuss this

File: README.md [hunk 3]
```diff
@@ -22,46 +22,74 @@
...
+|               postgres                | PostgreSQL database server for risk-profiler           | psql://localhost:5433                                                   | Uses port forwarding                                                                      |
+|                 redis                 | Redis cache for user 'session' data (roles)            | localhost:6379/tcp                                                      | None - required locally                                                                   |
+|              SQS (event)              | AWS SQS queue for events                               | http://localhost:4576<br>Name: event<br>(localstack)                    | EVENT_QUEUE_URL                                                                           |
```
Comment (kberec) [inline README.md:54]: Add SQS_ENABLED?

File: README.md [hunk 4]
```diff
@@ -22,46 +22,74 @@
...
+
+`Alternative way`
 
 Run redis as a local docker container on the default port of 6379 when running the app locally with tls disabled (the default)
 
-`docker run -p6379:6379 redis`
+```docker run -p6379:6379 redis```
 
 The other option is to run stunnel and port forward using the cloud platform guidance:
 `https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster`
```
Comment (kberec) [inline README.md:95]: I think there is a better guide in confluence how to set up up port forwarding

File: README.md [hunk 5]
```diff
@@ -72,38 +100,76 @@ REDIS_AUTH_TOKEN=<from the namespace secret>
...
+
+### Unit Tests
+To run the jest unit tests:
+
+```bash
+npm run test
+```
 
-`npm run lint`
+### Integration Tests
```
Comment (kberec) [inline README.md:157]: Let's discuss this as well

**Review by kberec — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (Olusola-Opencast) [reply README.md:51]: Resolved

**Review by kberec — APPROVED** [review]

**Pull request merged**